### PR TITLE
[MIG] web_widget_digitized_signature: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
- Updated js libraries

Standard migration

cc @Tecnativa

### CODE REVIEW

**Review by tbaden — APPROVED** [review]
tested in runbot

File: web_widget_digitized_signature/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright 2004-2010 OpenERP SA (<http://www.openerp.com>)
+# Copyright 2011-2015 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Web Widget Digitized Signature",
+    "version": "11.0.0.1.0",
```
Comment (tarteo) [inline web_widget_digitized_signature/__manifest__.py:8]: 11.0.1.0.0

File: web_widget_digitized_signature/static/src/js/digital_sign.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                'background-color': '#fff',
+                'height': '150',
+                'width': '550',
+                'clear': true
+            };
+            this.$(".signature").jSignature(sign_options);
+            this.$(".signature").focus();
+            this._setValue(false);
+        },
+        _on_save_sign: function(value_) {
```
Comment (tarteo) [inline web_widget_digitized_signature/static/src/js/digital_sign.js:62]: `value_` is never used

**Review by tarteo — APPROVED** [review]
LGTM, just some minor comments. I have some other comments but are out of scope for this PR.

File: web_widget_digitized_signature/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply web_widget_digitized_signature/__manifest__.py:8]: Done on later commit

**Pull request merged**